frontend/src/components/ui-extensions/enhanced-date-picker/date-picker.tsx

```
@@ -0,0 +1,247 @@
+        }
+      : undefined;
+  };
+
+  const updateDashboardDateRange = (
+    newRange: DateRange | undefined,
+    setDateRangeAndOption: (
+      option: DashboardDateRangeOptions,
+      date?: DashboardDateRange,
+    ) => void,
+  ) => {
+    if (newRange && newRange.from && newRange.to) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)